README.md

```diff
@@ -284,7 +300,7 @@ of all services that are accessible from your local machine
 
 ### ensure-kubectl
 Makes sure kubectl is installed and available for use. Customize the version
-by specifying the `KUBECTL_VERSION` envrionmental variable. Default: `v1.3.6`.
+by specifying the `KUBECTL_VERSION` envrionmental variable. Default: `v1.4.0`.
```

Comment:
Do you know if `glcoud` respects this environment variable when grabbing kubectl?